flecsi-sp/minimal/mesh.h

```diff
@@ -122,15 +161,21 @@ class minimal_mesh_t
   size_t
   indices(
     size_t index_space_id
+<<<<<<< HEAD
```

Comment:
As @charest pointed out to me yesterday, the build is likely using a docker image that has an installed `flecsi` that was built from the `master` branch, which is currently broken.  We'll need to either fix this in the docker container or change which branch we pull from for `flecsi`